packages/gatsby/src/commands/develop.ts

```diff
@@ -272,34 +272,35 @@ async function startServer(program: IProgram) {
   // Set up API proxy.
   const { proxy } = store.getState().config
   if (proxy) {
-    const { prefix, url } = proxy
-    app.use(`${prefix}/*`, (req, res) => {
-      const proxiedUrl = url + req.originalUrl
-      const {
-        // remove `host` from copied headers
-        // eslint-disable-next-line no-unused-vars
-        headers: { host, ...headers },
-        method,
-      } = req
-      req
-        .pipe(
-          got
-            .stream(proxiedUrl, { headers, method, decompress: false })
-            .on(`response`, response =>
-              res.writeHead(response.statusCode!, response.headers)
-            )
-            .on(`error`, (err, _, response) => {
-              if (response) {
+    proxy.forEach(({prefix, url}) => {
```

Comment:
What if only an object is set in the config?
Would still behave as an array?

Reply:
Object would be converted to array by joi